bm-persona/Dining/DiningDataSource/DiningHalls/DiningHall.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  DiningHall.swift
+//  bm-persona
+//
+//  Created by Kevin Hu on 3/14/20.
+//  Copyright © 2020 RJ Pimentel. All rights reserved.
+//
+
+import Foundation
+
+/**
+    `DiningHall` represents a single physical dining location.
+    Each hall contains the `DiningMenu`, Open & Close times for every `MealType`.
+*/
+#warning("TODO: Make DiningHall implement SearchItem and HasLocation")
+class DiningHall: HasOpenTimes {
+    
+    let name: String
```

Comment:
How are we planning to track whether a place is favorited?

Reply:
Probably another protocol. The old codebase used a `Favorable` protocol (required the boolean `isFavorited`) with a `FavoriteStore` class that managed saving / reading the object's state from the device.

Reply:
We can do this after initial release then